##### 2023-01-19 11:50:35.351061 #####
Traceback (most recent call last):
  File "site-packages\flask\app.py", line 2292, in wsgi_app
  File "site-packages\flask\app.py", line 1815, in full_dispatch_request
  File "site-packages\flask_restx\api.py", line 639, in error_router
  File "site-packages\flask\app.py", line 1718, in handle_user_exception
  File "site-packages\flask\_compat.py", line 35, in reraise
  File "site-packages\flask\app.py", line 1813, in full_dispatch_request
  File "site-packages\flask\app.py", line 1799, in dispatch_request
  File "site-packages\flask_login\utils.py", line 228, in decorated_view
  File "app\helpers.py", line 133, in decorated
  File "app\views\customize.py", line 676, in background_tasks
  File "app\tasks\__init__.py", line 37, in start_tasks
  File "app\tasks\base.py", line 38, in init
TypeError: __class__ assignment: 'QThreadBase' object layout differs from 'CacheTicketsAnnouncements'
##### 2023-01-19 11:51:48.805322 #####
Traceback (most recent call last):
  File "site-packages\flask\app.py", line 2292, in wsgi_app
  File "site-packages\flask\app.py", line 1815, in full_dispatch_request
  File "site-packages\flask_restx\api.py", line 639, in error_router
  File "site-packages\flask\app.py", line 1718, in handle_user_exception
  File "site-packages\flask\_compat.py", line 35, in reraise
  File "site-packages\flask\app.py", line 1813, in full_dispatch_request
  File "site-packages\flask\app.py", line 1799, in dispatch_request
  File "site-packages\flask_login\utils.py", line 228, in decorated_view
  File "app\helpers.py", line 133, in decorated
  File "app\views\customize.py", line 676, in background_tasks
  File "app\tasks\__init__.py", line 37, in start_tasks
  File "app\tasks\base.py", line 38, in init
TypeError: __class__ assignment: 'QThreadBase' object layout differs from 'CacheTicketsAnnouncements'
